addons/mail/security/ir.model.access.csv

```diff
@@ -15,14 +15,14 @@ access_mail_followers_system,mail.followers.system,model_mail_followers,base.gro
 access_mail_notification_portal,mail.notification.portal,model_mail_notification,base.group_portal,1,0,0,0
 access_mail_notification_user,mail.notification.user,model_mail_notification,base.group_user,1,1,1,0
 access_mail_notification_system, mail.notification.system,model_mail_notification,base.group_system,1,1,1,1
-access_mail_channel_all,mail.group.all,model_mail_channel,,1,0,0,0
-access_mail_channel_user,mail.group.user,model_mail_channel,base.group_user,1,1,1,0
-access_mail_channel_admin,mail.group.system,model_mail_channel,base.group_system,1,1,1,1
-access_mail_channel_member_public,mail.channel.member.public,model_mail_channel_member,base.group_public,1,0,0,0
-access_mail_channel_member_portal,mail.channel.member.portal,model_mail_channel_member,base.group_portal,1,1,1,1
-access_mail_channel_member_user,mail.channel.member.user,model_mail_channel_member,base.group_user,1,1,1,1
-access_mail_channel_rtc_session_all,mail.channel.rtc.session.all,model_mail_channel_rtc_session,,0,0,0,0
-access_mail_channel_rtc_session_system,mail.channel.rtc.session.system,model_mail_channel_rtc_session,base.group_system,1,1,1,1
+access_discuss_channel_all,mail.group.all,model_discuss_channel,,1,0,0,0

```

Comment:
please use this opportunity to remove this global rule

Reply:
I tried but this is breaking tests and it is out of scope of the PR, we need to merge the rename quickly to avoid conflicts.
I'll keep this rule in mind for next steps of the refactoring.